RestoreArray.java

```diff
@@ -0,0 +1,22 @@
+import java.util.Arrays;
+
+public class RestoreArray {
+    static int[] modifyArray(int[] arr) {
+        for (int i = 0; i < arr.length; i++) {
+            if (arr[i] < 0 && arr[i - 1] < arr[i + 1]) {
+                arr[i] = arr[i - 1] + 1;

```

Comment:
this method actually returns the same array it receives as an argument

Reply:
I run everything works

[5, 6, 1, 2, 6] - Original Array

[1, 2, 6, 5, 6] - Modified Array

Reply:
try to print original array after invoke method